### ISSUE TITLE
Vertical Temperature Slider Example: Duplicative screen reader text following label

### ISSUE THREAD

**Issue by jscholes** [w3c/aria-practices#3257]
In the [Vertical Temperature Slider Example](https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-temperature/), a textual representation of the current value is rendered inside the SVG before the element with `role="slider"`.  IN the following example markup, the text is "25.0°C" (the default at page load):

```
<div id="id-temp-label" class="label">Temperature</div>
<svg role="none" class="slider-group" ...>
	<text class="temp-value" x="28" y="35">25.0°C</text>
	<rect class="rail" x="60" y="47" width="8" height="300" rx="5" aria-hidden="true"></rect>
	<g role="slider" .../>
	...
/svg>
```

The `<text>` element with `class="temp-value"` is exposed to screen readers, meaning that users:
* Hear the value twice, once from this text and once from the `aria-valuetext` attribute on the semantic slider element; and
* have to move their screen reader cursor passed the text that is of no use to them.

For example, here's a speech log from NVDA with speaking of command keys enabled, starting from just before the label:

> down arrow
> Temperature
> down arrow
> 25.0°C
> down arrow
> slider    25.0 degrees Celsius

In addition, it's not clear why the slider visually shows "°C" as the unit of measurement, while the `aria-valuetext` attribute explicitly expands it to "degrees Celsius".  Either all users should be expected to understand the common "°C" notation, or it should be spelled out for everyone.  Giving screen reader users more explanation encourages inequitable over-labelling, has a disproportionate impact in braille, etc.

CC @IsaDC, @mcking65

**Comment by jongund** [comment]
Updated [Vertical Slider Preview](https://deploy-preview-400--aria-practices.netlify.app/aria/apg/patterns/slider/examples/slider-temperature/) that fixes the redundant speaking of the temperature value
